mods/psionics/system/psionics/mob/borer_power.dm

```diff
@@ -2,6 +2,11 @@
 	var/image/aura_image
 
```

Comment:
Maybe this file should be included if borer mod is included only

Reply:
I can't see a way to reliably do that since the include order is not predictable. :(